src/aics-image-viewer/shared/constants.ts

```diff
@@ -135,19 +153,29 @@ export const getDefaultViewerState = (): ViewerState => ({
   cameraState: USE_VIEW_MODE_DEFAULT_CAMERA,
 });
 
-export const getDefaultChannelState = (): ChannelState => ({
-  name: "",
-  volumeEnabled: false,
-  isosurfaceEnabled: false,
-  colorizeEnabled: false,
-  colorizeAlpha: 1.0,
-  isovalue: 128,
-  opacity: 1.0,
-  color: [226, 205, 179] as ColorArray,
-  useControlPoints: false,
-  ramp: [0, TFEDITOR_MAX_BIN],
-  controlPoints: [
-    { x: 0, opacity: 0, color: [255, 255, 255] },
-    { x: 255, opacity: 1, color: [255, 255, 255] },
-  ],
-});
+/**
+ * Returns the default channel state as a new object. If an index is provided, uses the default
+ * color preset for that index.
+ * @param index Optional channel index to use for the color preset.
+ * @returns a default ChannelState object.
+ */
+export const getDefaultChannelState = (index: number = 0): ChannelState => {
+  const color = PRESET_COLORS_0[index] || PRESET_COLORS_0[0];
```

Comment:
nit: how about using PRESET_COLORS_0[index % num_of_preset_colors] ?

Reply:
If you load something with a lot of channels, the default behavior is that it doesn't repeat the color sequence. Everything defaults to beige.

Reply:
yeah, the suggestion here is to change that, on the assumption that it would actually be an improvement :)

Reply:
Fixed! Change visible here: http://localhost:9020/viewer?url=https%3A%2F%2Fuk1s3.embassy.ebi.ac.uk%2Fidr%2Fzarr%2Fv0.4%2Fidr0076A%2F10501752.zarr